simpa_examples/minimal_optical_simulation.py

```diff
@@ -139,7 +139,7 @@ def __init__(self):
                                                         VOLUME_PLANAR_DIM_IN_MM/2, 0]))
         self.set_detection_geometry(sp.LinearArrayDetectionGeometry())
         self.add_illumination_geometry(sp.SlitIlluminationGeometry(slit_vector_mm=[20, 0, 0],
-                                                                   direction_vector_mm=[0, 0, 5]))
```

Comment:
This change does not matter because of the normalisation, correct?

Reply:
Yes, it should not matter, but I think as example, we should do it normalised to 1 as "best practice"

Reply:
Agreed!